buildenv

```diff
@@ -0,0 +1,44 @@
+export ZOPEN_DEV_URL="https://github.com/libarchive/libarchive.git"
```

Comment:
Can you add the bump line for automatic updating? E.g https://github.com/zopencommunity/gitport/blob/main/buildenv#L5-L12